.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules
 .env
 .idea/
 venv/
+__pycache__/
```

Comment:
`PYTHONDONTWRITEBYTECODE=1` for the win.

Reply:
This setup doesn't really have the tools to use that setting automatically as it's just a script to run, and doesn't have something like a task runnner/docker container to run in. But feel free to apply it to your shell!

Reply:
I think I even set that globally in windows... the number of pycache files and dirs was too damn high